Adding React Native Versions and Lifecycle support policy  to readme (#39)
reactwg/react-native-releases: README.md

# React Native Releases

This is a discussion focussed repository for coordinating React Native releases.

> 
> ## Contents
> - [React Native Version and Lifecycle support](#-react-native-version-and-lifecycle-support)
> 
> 
> 

Reviewer: ```suggestion

```

maybe remove this part? a bit redundant

At this point in time, we are prioritizing issues that are related to:

- Latest version of React Native and two previous minor series.
- Use of the New Architecture
- Use of the Hermes Engine

### Cherry-Pick Requests

We’re **accepting cherry-pick requests** for the currently supported versions. A cherry-pick request is the request to include a fix in one of the supported versions and release a new point release with it.

Cherry-Pick requests should be submitted via the [React Native Releases Working Group](https://github.com/reactwg/react-native-releases/discussions/categories/patches), in the corresponding discussion thread.

Please note that each cherry-pick request will be assessed and approved individually. Cherry-Pick requests against unsupported versions will be rejected unless they involve security issues.

### Security Issues

Security issues should follow our issue report policy [described here](https://reactnative.dev/contributing/overview#security-bugs).
Reports for security issues on unsupported versions will handed on case by case basis when deciding if a new version needs to be released.

> ### Licence

Reviewer: ```suggestion
### License
```

> React Native is open-source software distributed under the **MIT license**. Software is provided "as is" without warranty.

Reviewer: ```suggestion
React Native is open-source software distributed under the [**MIT license**](https://github.com/facebook/react-native/blob/main/LICENSE).
```
